src/main/scala/cromwell/binding/RuntimeAttributes.scala

```diff
@@ -49,7 +49,7 @@ object RuntimeAttributes {
 
   val LocalDiskName = "local-disk"
 
-  val LocalizationDisk = LocalDisk(LocalDiskName, DiskType.LOCAL).toDisk
+  val LocalizationDisk = LocalDisk(LocalDiskName, DiskType.SSD).toDisk
```

Comment:
Can you explicitly add the default size of the SSD here too? I believe 10GB is the minimum.